src/components/CountryDetail.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable no-undef */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly avoid skipping the linters, they are in place to help you write more clean code. it would be a good idea to fix all the linter errors in your app instead of skipping them